bin/exec.js

```diff
@@ -146,71 +146,80 @@ try {
   cleanUp(msg + err);
 }
 
-// Run modified testrpc with large block limit, on (hopefully) unused port.
```

Comment:
Dunno why diff didn't see this, but most of the change in here is really just an indent level. Try dedenting the Promise clauses and you should see that.